### PR TITLE
changes to color scheme docstring

### PR DESCRIPTION
**Pull request by michellemho**
When using 'color' argument in Layer Class as a dictionary, the `scheme` key does not take a string type value. It requires a dictionary of the styling scheme. Docstring is changed in this PR to reflect this.

Attempting to color by numeric column by quantiles using a string for `scheme` fails:
```Python
# numeric
cc.map(layers=[BaseMap(),
               Layer('all_month_3',
                     color = {'column':'mag','scheme':'Sunset','bin_method':'quantiles','bins':5})], 
       interactive=True)
```
But this syntax works
```Python
cc.map(layers=[BaseMap(),
               Layer('all_month_3',
                     color = {'column':'mag','scheme':{'bin_method': 'quantiles', 'bins': 5, 'name': 'Sunset'}})],
       interactive=True)
```
And this works as well:

```Python
from cartoframes.styling import sunset
cc.map(layers=[BaseMap(),
               Layer('all_month_3',
                     color = {'column':'mag','scheme':sunset(5)})],
       interactive=True)
```

### CODE REVIEW

File: cartoframes/layer.py [hunk 1]
```diff
@@ -184,15 +184,10 @@ class QueryLayer(AbstractLayer):
...
+            - scheme (dict, optional): Dictionary definition of color scheme from
+              styling module of cartoframes that exposes `CartoColors
               <https://github.com/CartoDB/CartoColor/wiki/CARTOColor-Scheme-Names>`__.
-              Defaults to `Mint`.
-            - bin_method (str, optional): Quantification method for dividing
-              data range into bins. Must be one of: ``quantiles``, ``equal``,
-              ``headtails``, or ``jenks``. Defaults to ``quantiles``.
-            - bins (int, optional): Number of bins to divide data amongst in
-              the `bin_method`. Defaults to 5.
+              Defaults to mint scheme.
```
Comment (andy-esch) [inline cartoframes/layer.py]: Could you also add something to the effect of "..., or a scheme from the styling module such as `styling.sunset(7)`"?

Comment (michellemho) [reply cartoframes/layer.py]: done!

File: cartoframes/layer.py [hunk 2]
```diff
@@ -184,15 +184,11 @@ class QueryLayer(AbstractLayer):
...
 
             - column (str): Column to base coloring from.
-            - scheme (str, optinal): Color scheme from
-              `CartoColors
+            - scheme (dict, optional): Dictionary definition of color scheme,
+              or a scheme such as `styling.sunset(7)` from the
+              styling module of cartoframes that exposes `CartoColors
               <https://github.com/CartoDB/CartoColor/wiki/CARTOColor-Scheme-Names>`__.
-              Defaults to `Mint`.
-            - bin_method (str, optional): Quantification method for dividing
```
Comment (andy-esch) [inline cartoframes/layer.py]: These other keys are still possible. A user could do
```python
cc.map(layers=Layer(..., color={'column': 'axolotl',
                                'scheme': dict(bin_method='quantiles', bins=7, name='Vivid')}))
```

And actually, there might be a hidden 'colors' key somewhere you can fill custom values!

**Comment by andy-esch** [comment]
Thanks for finding this!

**Comment by andy-esch** [comment]
@michellemho I had to quick-commit some work here before they took my computer away for repair. I'll wrap up the work soon.

**Pull request merged**